src/Utilities/time.ts

```diff
@@ -2,9 +2,9 @@ export const parseYYYYMMDDToDate = (val: string) =>
   val ? new Date(`${val}T00:00:00`) : new Date('');
 
 export const yyyyMMddFormat = (date: Date) =>
-  `${date.getFullYear()}-${(date.getMonth() + 1)
```

Comment:
The only problem I see is the same behaviour you've mentioned in https://github.com/osbuild/image-builder-frontend/pull/2526/files#r1815188998, but the other way around. Without the UTC the picked date seems accurate, but with UTC it's a always set as the day before. Could it be some weird time zoney thing? :thinking:

Reply:
thank you for the review @regexowl ! 

ah i see what you mean, when i changed my system time to Brno i see the same issue. since dateParse and dateFormat have [defaults](https://www.patternfly.org/components/date-and-time/date-picker/#datepicker) that look similar to the helper functions, i removed those fields in the DatePicker and formatted the date for the value in the picker to just remove the timestamp. it looks ok now on my end for both Brno and New York timezones. let me know what you think 🤔

Reply:
Excellent, that did the trick! Thank you!